src/content/en/2021/pwa.md

```diff
@@ -17,4 +17,597 @@ featured_stat_3: TODO
+The `standalone` mode makes installed PWAs open without any browser UI element, making it "feel like an app". The chart shows that the most sites with a service worker and manifest uses this value: 74.83% on desktop and 79.02% on mobile.
+
+### Manifests preferring native
+
+Finally we'll analyze `prefer_related_applications`. If the value of this property is set to `true`, the browser might suggest installing one of the related applications instead of the web app.
+
+{{ figure_markup(
+  image="pwa-manifests-preferring-native-app.png",
+  caption="Manifests preferring native app.",
+  description="Bar chart showing PWA manifests preferring native app with 97.92% of desktop PWA pages, and 98.03% of mobile PWA pages having this set to FALSE, and only 1.86% of desktop pages, and 1.79% of mobile pages having it set to TRUE.", chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTHHnqNdpRUjoeTfsN9_irK57PvZn_Q2X842RLl-RL4ibWmZFvO-S1x35PjVE3-xUlHFS_Zurd22rOq/pubchart?oid=1882009659&format=interactive",
+  sheets_gid="1552027039",
+  sql_file="manifests_preferring_native_apps.sql"
+  )
+}}
+
+`prefer_related_applications` appears only in 6.87% of desktop and 7.66% of sites.The chart shows that 97.92% of desktop and 93.03% of mobile sites that defined this property have a value of `false`. This indicates that most PWA developers prefer to offer the PWA experience rather than a native app.
+
+Despite the fact that the vast majority of PWA developers prefer promoting their PWA experiences to native applications, some well known PWAs (like Twitter), still prefer recommending the native app over the PWA experience. This might be due to a preference of the teams building these experiences, or some specific business needs (lack of some API in the web).
+
+<p class="note">Note: Instead of making this decision statically at configuration, developers can also <a hreflang="en" href="https://web.dev/define-install-strategy/">create more dynamic heuristics</a> to promote an experience, for example, based on the user's behavior or other characteristics (device, connection, location, etc).</p>
+
+### Top manifest categories
+
+In last year's PWA chapter we included a section about <a hreflang="en" href="../2020/pwa#top-manifest-categories">manifest categories</a>, showing the percentage of PWAs per industry, based on the <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Web/Manifest/categories">manifest categories</a> property.
+
+This year we decided not to rely on this property to determine how many PWAs of each category are out there, since the usage of this property is rather low: <a hreflang="en" href="https://docs.google.com/spreadsheets/d/16AkIdDBBkCR5Kgb7kyfYvnNLQBu23Vsh7MUSFHW9RtA/edit#gid=137691358">only  0.68% of mobile and 0.76% of desktop manifests have this property</a>.
+
+In the next section, we'll use an external analysis to get an approximation of this.
+
+## PWA use cases by industry
```

Comment:
Yes! This makes sense. I didn't know how to make the Mobsted analysis fit into Manifests (since they are essentially different things), and that's why I used a separate one. This would allow us to put both on the same place.